__tests__/api/token/temporary.test.ts

```diff
@@ -86,10 +94,10 @@ describe("TemporaryBearerToken tests", () => {
     });
 
     await temporary(req, res);
-    expect(res.statusCode).toEqual(403);
+    expect(res.statusCode).toEqual(400);
   });
 
-  it("throws error with invalid bearer token", async () => {
+  it("throws error with invalid form access token", async () => {
```

Comment:
"form access" and "bearer" tokens are the same thing. The term "form access" was introduced from a UI perspective, with the assumption that the average user wouldn't be familiar with the term "bearer token".

No need to change it back, just a comment.